css/styles.css

```diff
@@ -113,3 +121,90 @@ header, footer {
 .tt-cursor, .tt-suggestion:hover {
     background-color: #efefef;
 }
+
+/* improvements by YuriBrunetto */
```

Comment:
I appreciate you wanting to keep your changes separate, but let's try to introduce some logical ordering. I like that you put the `body` rule above the `header, footer` rule. For example, let's move the `header` and `footer` rules below the `header, footer` rule.